packages/legend-query/src/components/QueryBuilderFilterPanel.tsx

```diff
@@ -514,6 +557,16 @@ const QueryBuilderFilterTreeNodeContainer = observer(
             </div>
           </div>
           <div className="query-builder-filter-tree__node__actions">
+            {node instanceof QueryBuilderFilterTreeConditionNodeData && (
```

Comment:
This is quite a weird thing for user to be able to do, that's why I never exposed it, is there a good reason why we do it here?

Reply:
In what way is that weird? Where else would you reset the value spec value

Reply:
Change to reset